### PR TITLE
Update kinto dependencies to not force postgresql and allow kinto extra_dependencies

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by leplatrem** [comment]
Please rename your PR (and thus upcoming merge commit) if FxA is concerned :)

**Comment by Natim** [comment]
Done :+1:

File: docs/installation.rst [hunk 1]
```diff
@@ -14,12 +14,7 @@ Run locally
 For development
 ---------------
 
-By default, *Kinto* persists the records and internal cache in a PostgreSQL
-database.
-
-The default configuration will connect to the ``postgres`` database on
-``localhost:5432``, with user/password ``postgres``/``postgres``.
-See more details below about installation and setup of PostgreSQL.
+By default, *Kinto* persists the records, permissions and internal cache in redis.
```
Comment (leplatrem) [inline docs/installation.rst]: Please remove this changes from this PR (out of scope), it will be adressed in #90 or #95

File: kinto/__init__.py [hunk 2]
```diff
@@ -14,9 +14,6 @@
 
 
 DEFAULT_SETTINGS = {
-    'cliquet.cache_backend': 'cliquet.cache.memory',
-    'cliquet.permission_backend': 'cliquet.permission.memory',
-    'cliquet.storage_backend': 'cliquet.storage.memory',
```
Comment (leplatrem) [inline kinto/__init__.py]: ditto

File: docs/installation.rst [hunk 3]
```diff
@@ -60,7 +55,15 @@ Recommended settings for production are listed :ref:`in a dedicated section
...
 
- (*requires PostgreSQL 9.3 or higher*).
+ (*requires PostgreSQL 9.4 or higher*).
+
+Install Python dependencies
+---------------------------
+
+::
+
+    pip install kinto[postgresql]
```
Comment (leplatrem) [inline docs/installation.rst]: I think it requires quotes

**Comment by Natim** [comment]
Updated.

File: Makefile [hunk 4]
```diff
@@ -18,6 +18,9 @@ $(INSTALL_STAMP): $(PYTHON)
 	$(VENV)/bin/pip install -Ue .
 	touch $(INSTALL_STAMP)
 
+install-postgres: $(INSTALL_STAMP)
+	$(VENV)/bin/pip install -Ue .[postgresql]
```
Comment (leplatrem) [inline Makefile]: This does not install postgresql deps in Travis
https://travis-ci.org/mozilla-services/kinto/jobs/69301205#L244

**Pull request merged**